src/components/SVGIcons/IconArrowEnterFilled.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react'
+import { ISVGIconProps } from './types'
+import classNames from 'classnames'
+
+export const IconArrowEnterFilled = ({
+  size,
+  type,
+  className = '',
+  onClick,
+  refHandler,
+  id,
+  dataId
```

Comment:
@ManukTagushyan jan maybe also have ...restProps and send those to svg?

Reply:
At this time, I think there's no need to do that. But when we need it, we can add it to the generate_svg_component.js file.